benchmarks/gaussian_localisation/gaussian_test.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import dask
+import glob
+import re
+import sys
+import os
+
+import deepblink as pink
+import numpy as np
+import scipy.optimize as opt
+
+sys.path.append("../")
+from util import delayed_results
+
+EPS = 1e-4
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--datasets")
+    parser.add_argument("--models")
+    args = parser.parse_args()
+    return args
+
+
+@dask.delayed
+def delayed_normalize_pink(image):
+    return pink.data.normalize_image(image)
+
+
+def gauss_2d(xy, amplitude, x0, y0, sigma_xy, offset):
+    """2D gaussian."""
+    x, y = xy
+    x0 = float(x0)
+    y0 = float(y0)
+    gauss = offset + amplitude * np.exp(
+        -(
+            ((x - x0) ** (2) / (2 * sigma_xy ** (2)))
+            + ((y - y0) ** (2) / (2 * sigma_xy ** (2)))
+        )
+    )
+    return gauss
+
+
+def gauss_single_spot(
+    image: np.ndarray, c_coord: float, r_coord: float, crop_size: int
+):
+    """Gaussian prediction on a single crop centred on spot."""
+    start_dim1 = np.max([int(np.round(r_coord - crop_size // 2)), 0])
+    if start_dim1 < len(image) - crop_size:
+        end_dim1 = start_dim1 + crop_size
+    else:
+        start_dim1 = len(image) - crop_size
+        end_dim1 = len(image)
```

Comment:
I guess `min(len(image), start_dim1+crop_size)` would also work, but this is fine.